INSTALLATION.md

```diff
@@ -63,11 +63,13 @@ First you need a local copy of `talawa-admin`. Run the following command in the
       ![Image of user's clone](public/markdown/images/install2.png)
 
    1. Clone the repository to your local computer (replacing the values in `{{}}`):
+
       ```bash
       $ git clone https://github.com/{{YOUR GITHUB USERNAME}}/talawa-admin.git
       cd talawa-admin
       git checkout develop
       ```
+
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project